calendar-bundle/src/Resources/contao/modules/ModuleEventlist.php

```diff
@@ -358,32 +361,28 @@ protected function compile()
 			$objTemplate->addBefore = false;
 
 			// Add an image
-			if ($event['addImage'] && $event['singleSRC'])
+			if ($event['addImage'] && null !== ($figureBuilder = $this->getFigureBuilderIfResourceExists($event['singleSRC'])))
 			{
-				$objModel = FilesModel::findByUuid($event['singleSRC']);
-
-				if ($objModel !== null && is_file($projectDir . '/' . $objModel->path))
-				{
-					if ($imgSize)
-					{
-						$event['size'] = $imgSize;
-					}
+				/** @var CalendarEventsModel $eventModel */
+				$eventModel = CalendarEventsModel::findByPk($event['id']);
 
-					$event['singleSRC'] = $objModel->path;
-					$this->addImageToTemplate($objTemplate, $event, null, null, $objModel);
+				$figure = $figureBuilder
+					->setSize($eventModel->size ?: $this->imgSize ?: null)
```

Comment:
Same here.

Reply:
Fixed in 1db55b3d80534f519efddf28902881709787bf51